src/config.rs

```diff
@@ -659,6 +659,22 @@ impl Cfg {
     pub(crate) fn find_or_install_override_toolchain_or_default(
         &self,
         path: &Path,
+    ) -> Result<(Toolchain<'_>, Option<OverrideReason>)> {
+        self.find_toolchain(path, true)
+    }
+
+    #[cfg_attr(feature = "otel", tracing::instrument(skip_all))]
+    pub(crate) fn find_override_toolchain_or_default(
+        &self,
+        path: &Path,
+    ) -> Result<(Toolchain<'_>, Option<OverrideReason>)> {
+        self.find_toolchain(path, false)
```

Comment:
this is just find_toolchain with a create option of false. I don't think the new function is worth it.